src/Tribe/Ignored_Events.php

```diff
@@ -0,0 +1,526 @@
+<?php
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+if ( ! class_exists( 'Tribe__Events__Ignored_Events' ) ) {
+	/**
+	 * Ignored Events are fully powered by this class
+	 */
+	class Tribe__Events__Ignored_Events {
+		public static $ignored_status = 'tribe-ignored';
```

Comment:
I know you're not a fan of class constants, but these _could_ be class constants instead vs public, overwriteable fields #justSaying